lib/mocha.js

```diff
@@ -99,22 +99,6 @@ function Mocha(options) {
   if (options.slow) {
     this.slow(options.slow);
   }
-
-  this.suite.on('pre-require', function(context) {
-    exports.afterEach = context.afterEach || context.teardown;
-    exports.after = context.after || context.suiteTeardown;
-    exports.beforeEach = context.beforeEach || context.setup;
-    exports.before = context.before || context.suiteSetup;
-    exports.describe = context.describe || context.suite;
-    exports.it = context.it || context.test;
-    exports.setup = context.setup || context.beforeEach;
-    exports.suiteSetup = context.suiteSetup || context.before;
-    exports.suiteTeardown = context.suiteTeardown || context.after;
-    exports.suite = context.suite || context.describe;
-    exports.teardown = context.teardown || context.afterEach;
-    exports.test = context.test || context.it;
-    exports.run = context.run;
-  });
 }
```

Comment:
What this code is doing is making these functions available via `require('mocha').describe` so that you can do `import { describe, it } from 'mocha'`. So yes, it's related, but it's not the part of the code that is setting up the globals.